<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>928</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mendaftarkan TIcket</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="bgWindow">
   <property name="styleSheet">
    <string notr="true">#bgWindow{
background-color: qlineargradient(spread:pad, x1:0, y1:0.961, x2:1, y2:0, stop:0 rgba(33, 137, 205, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
   </property>
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>490</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(0, 0, 0);
color:rgba(255,255,255,210);
border-radius:15px;
font: 12pt &quot;MS Shell Dlg 2&quot;; color:rgb(255, 255, 255);
}

QPushButton:hover{
background-color:qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(150, 123, 111, 219), stop:1 rgba(85, 81, 84, 226));
}

QPushButton:pressed{
padding-left:5px;
padding-top:5px;
background-color:rgba(150, 123, 111, 255);
};</string>
    </property>
    <property name="text">
     <string>Kirim</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>490</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(0, 0, 0);
color:rgba(255,255,255,210);
border-radius:15px;
font: 12pt &quot;MS Shell Dlg 2&quot;; color:rgb(255, 255, 255);
}

QPushButton:hover{
background-color:qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(150, 123, 111, 219), stop:1 rgba(85, 81, 84, 226));
}

QPushButton:pressed{
padding-left:5px;
padding-top:5px;
background-color:rgba(150, 123, 111, 255);
};</string>
    </property>
    <property name="text">
     <string>Kembali</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Pusat Bantuan</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Judul</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Deskiripsi</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="deksripsiTextbox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>831</width>
      <height>70</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Masukan Deskripsi</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>430</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Skala Prioritas</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Attachment URL</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="URLTextBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>831</width>
      <height>70</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Masukan URL</string>
    </property>
   </widget>
   <widget class="QComboBox" name="priorityBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>430</y>
      <width>171</width>
      <height>29</height>
     </rect>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="judulTextbox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>831</width>
      <height>41</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Masukan Judul</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
